kansa/lib/mail.js

```diff
@@ -32,32 +29,29 @@ const mailRecipient = (email, res) => {
       name = names.join(', ')
   }
   const attending = res
-    .filter(r => {
-      switch (r.membership) {
-        case 'Supporter': return false
-        case 'NonMember': return !!r.daypass
-        default: return true
-      }
-    })
+    .filter(r => r.badge || r.daypass)
     .map(({ id, name }) => ({ id, name }))
   const hugo_members = res
-    .filter(r => r.hugo_nominator || r.hugo_voter)
+    .filter(r => r.hugo_nominator || r.wsfs_member)
     .map(({ id, name }) => ({ id, name }))
   return {
     attending,
     email: res[0].email,
     hugo_members,
     key: res[0].key,
-    membership: mt[mi],
+    membership: res[0].membership, // FIXME
```

Comment:
What needs fixing here? Could you describe it in this comment, or just fix it ;-).

Reply:
Ah, sorry, this is #49.

Reply:
Fair enough. How about just putting that in the comment?

```
// FIXME will come back to this for issue #49
```